src/desktop/apps/auction_reaction/server.tsx

```diff
@@ -3,5 +3,6 @@ import { bidderRegistration } from "./routes"
 
 export const app = express()
 
+app.get("/auction-faq", bidderRegistration)
```

Comment:
the `bidderRegistration` function is already has a pretty clear responsibility, so I would be hesitant to add another here. Can we introduce a new "route" `auctionFAQ` that is _just_ responsible for rendering this new reaction component?

Reply:
I'll add this, but it's a lot of repeated code which combining the routes solved.

Reply:
Combining the two routes saves a lot of repeated code so I'm going to keep it the way it is, I'll rename the function, as Chris suggested.